djautotask/api.py

```diff
@@ -0,0 +1,89 @@
+import requests
+from requests.exceptions import ConnectTimeout, Timeout, ReadTimeout, SSLError
+from io import BytesIO
+import suds.transport as transport
+from atws.wrapper import AutotaskAPIException, ResponseQuery
+from atws import wrapper, connection
+
+from django.conf import settings
+from djautotask.utils import DjautotaskSettings
+
+
+def init_api_connection(**kwargs):
```

Comment:
I didn't include this as a class method on AutotaskRequestsTransport. Didn't make sense to me because `init_api_connection` returns a `Wrapper` class with a reference to `AutotaskRequestsTransport`.

Reply:
Oh yeah, that makes sense. Looks good to me.